backend/api/Areas/Organizations/Models/Organization/OrganizationModel.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Pims.Api.Models.Contact;
+
+namespace Pims.Api.Areas.Organizations.Models.Organization
+{
+    /// <summary>
+    /// Provides a contact-oriented organization model.
+    /// </summary>
+    public class OrganizationModel
+    {
+        #region Properties
+        /// <summary>
+        /// get/set - The organization's id.
+        /// </summary>
+        public long Id { get; set; }
+
+        /// <summary>
+        /// get/set - The concurrency row version.
+        /// </summary>
+        /// <value></value>
+        public long RowVersion { get; set; }
+
+        /// <summary>
+        /// get/set - The organization's disabled status flag.
+        /// </summary>
+        public bool IsDisabled { get; set; }
+
+        /// <summary>
+        /// get/set - The organization's name.
+        /// </summary>
+        [Required]
```

Comment:
Does this change anything in the API model?

Reply:
not really following, sorry. The only required field on this model is the org name. That is reflected here with a data annotation. The effect is that the API expects the json to have at least that property set.

This is standard MVC/webapi. If the controller receives json that doesn't pass validation it rejects the operation

Reply:
right, but this is not running the mvc validations. We should remove it since it does not seem to be doing anything for us atm

Reply:
could you please clarify why is not running validation. In my experience it does run by default (not by our code)

Reply:
As per documentation [here](https://www.pragimtech.com/blog/blazor/rest-api-model-validation/) and [here](https://docs.microsoft.com/en-us/aspnet/core/web-api/?view=aspnetcore-6.0#automatic-http-400-responses) we are using model validation automatically in our project as all controllers are annotated with `[ApiController]` 

I didn't know this - I just noticed that our controllers were validating the input model and took advantage of that